apps/pie-microsite/src/assets/img/designers/getting-started/libraries-wheel.svg

```diff
@@ -0,0 +1,26 @@
+<svg width="512" height="512" viewBox="0 0 512 512" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If you run this through SVGOMG I think you can trim it down by 80+%

Reply:
that's a fair concern, however, running the SVG through SVGOMG only managed to trim it down by 3.6% and it seems it broke its rendering. could you please try it out maybe I didn't do it correctly? 

also worth mentioning is that we will be re-evaluating our images/assets format in the near future and we might end up getting rid of all SVGs in favor of webp if I am not mistaken.

Reply:
With number and transform precision set to 3 I can get it reduced by ~90% with no visual differences

![2023-02-01 09_02_16](https://user-images.githubusercontent.com/26894168/215998117-f0dde983-f01e-4439-8d85-c0f5f91d05be.png)

Reply:
updated 🎑